functors task (1)/student/src/impl.cpp

```diff
@@ -1,2 +1,16 @@
 #include "impl.h"
 
+static constexpr BeerBrand kBeerBrandsOrdered[] = {
+    BeerBrand::HoeGaarden,
+    BeerBrand::Corona,
+    BeerBrand::Carlsberg,
+    BeerBrand::Bud,
+    BeerBrand::ZlataPraha,
+    BeerBrand::Leffe,
+};
+static constexpr auto kBeerBrandsOrderedSize = sizeof(kBeerBrandsOrdered) / sizeof(kBeerBrandsOrdered[0]);
+
+BeerBrand BeerOrganizer::operator()()
+{
+    return kBeerBrandsOrdered[i++ % kBeerBrandsOrderedSize];
```

Comment:
everything is much simpler:
we have a guarantee about enum values order here

you shouldn't use array

Reply:
That's true. Array removed.